diff --git a/src/MetatagManager.php b/src/MetatagManager.php
index bafd6f9..efbb0cc 100644
--- a/src/MetatagManager.php
+++ b/src/MetatagManager.php
@@ -550,7 +550,7 @@ class MetatagManager implements MetatagManagerInterface {
         $tag->setValue($value);
 
         // Obtain the processed value.
-        $processed_value = PlainTextOutput::renderFromHtml(htmlspecialchars_decode($this->tokenService->replace($tag->value(), $token_replacements, ['langcode' => $langcode])));
+        $processed_value = htmlspecialchars_decode($this->tokenService->replace($tag->value(), $token_replacements, ['langcode' => $langcode]));
 
         // Now store the value with processed tokens back into the plugin.
         $tag->setValue($processed_value);
