diff --git a/src/SchemaMetatagManager.php b/src/SchemaMetatagManager.php
index 88fa97a..526d32b 100644
--- a/src/SchemaMetatagManager.php
+++ b/src/SchemaMetatagManager.php
@@ -169,7 +169,7 @@ class SchemaMetatagManager implements SchemaMetatagManagerInterface {
    */
   public static function explode($value) {
     if (is_string($value)) {
-      $value = explode(',', $value);
+      $value = explode('::', $value);
     }
     if (is_array($value)) {
       $value = array_map('trim', $value);
