diff --git a/src/SchemaMetatagManager.php b/src/SchemaMetatagManager.php
index 6103e84..52d0f98 100644
--- a/src/SchemaMetatagManager.php
+++ b/src/SchemaMetatagManager.php
@@ -181,7 +181,7 @@ public static function countNumericKeys($item) {
    */
   public static function explode($value) {
     if (is_string($value)) {
-      $value = explode(',', $value);
+      $value = explode('::', $value);
     }
     if (is_array($value)) {
       $value = array_map('trim', $value);
